api/proto/teleport/dbobject/v1/dbobject.proto

```diff
@@ -0,0 +1,52 @@
+// Copyright 2023 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package teleport.dbobject.v1;
+
+import "teleport/header/v1/metadata.proto";
+
+option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/dbobject/v1;dbobjectv1";
+
+// DatabaseObject represents a database object that can be imported into Teleport.
```

Comment:
What is database object? An example could go a long way. Please document.

Reply:
This is and other pieces of information are covered in depth in the RFD. I'd rather keep a single source of truth here, but I can see an argument for adding a little more detail to the godocs.

Reply:
Done.

Reply:
That is already very helpful, thanks.